Dockerfile.redhat

```diff
@@ -20,6 +20,8 @@ ARG PKG_IMAGE=pkg
 ARG RELEASE_BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.7
 
 FROM $BASE_IMAGE as base_build
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.7
+ENV BASE_IMAGE=${BASE_IMAGE}
```

Comment:
Why do we need this ENV? It is not used

Reply:
Used to determine python version in pybind.